contracts/ControllerV1.sol

```diff
@@ -407,8 +407,6 @@ contract ControllerV1 is
 
         // if module is already disabled, the safe must unset these manually
         if (isSafeModuleEnabled(safe)) {
-            // remove controller as guard
-            setSafeGuard(safe, address(0));
             // remove module and handle reverse registration clearing.
             disableModule(
```

Comment:
can we update the `disableModule` to use `setSafeGuard` instead of replicating the logic internally